### PR TITLE
Prevent warning during server headers fetching (Telemetry); fixes #2856

### LINKED ISSUE #2856: PHP Warning on telemetry example page
During v9.2 installation process one can see an example of telemetry data.

Clicking the "_See what would be sent..._" link opens a popin of URL https://glpi.example.com/ajax/telemetry.php

Where the following error is displayed on the first line (note: the telemetry data example are present):

> PHP Warning: get_headers(http://localhost/glpi/): failed to open stream: HTTP request failed! in .../inc/telemetry.class.php at line 146

It looks like an (bad) assumption is done on path to GLPI instance: it thinks it is installed in a sub-directory "/glpi".

Using:
* GLPI v9.2
* PHP v5.6.30 (with Zend OPcache v7.0.6)
* Apache v2.4.10

### PR DESCRIPTION
**Pull request by cedric-anne**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

On some edge cases, even if hostname can be resolved, request to the server may fail.

### CODE REVIEW

**Review by trasher — APPROVED** [review]

**Review by orthagh — APPROVED** [review]

File: inc/telemetry.class.php [hunk 1]
```diff
@@ -147,19 +147,18 @@ static public function grabWebserverInfos() {
       $engine  = '';
       $version = '';
 
-      // check if host is present (do no throw php warning in contrary of get_headers)
-      if (filter_var(gethostbyname(parse_url($CFG_GLPI['url_base'], PHP_URL_HOST)),
-          FILTER_VALIDATE_IP)) {
+      // Try to ge headers only if hostname can be resolved
```
Comment (g-bougard) [inline inc/telemetry.class.php:150]: I see a little typo there: "ge" should be "get"

**Review by trasher — APPROVED** [review]

**Review by orthagh — APPROVED** [review]
